schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals(
     date_of_birth date,
     escape_attempts integer,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species varchar(250)
```

Comment:
- [ ] As mentioned by the previous reviewer, you should use `ALTER TABLE` query to add the species column like so 👇🏾 

```sql
   ALTER TABLE animals ADD species varchar(255);
```

Please remove line 10 and use the `alter` keyword as demonstrated above.